### PR TITLE
Update changelog for 3.7.10 release

### PR DESCRIPTION
**Pull request by pepone**

### CODE REVIEW

File: CHANGELOG-3.7.md [hunk 1]
```diff
@@ -8,87 +8,118 @@ We recommend that you use the release notes as a guide for migrating your
...
+
+These are the major changes since Ice 3.7.9.
+
+## C++ Changes
+
+- Fixes failures building from source with G++ 13  https://github.com/zeroc-ice/ice/issues/1489
+
+- Fixed the C++11 library names produced by Windows build https://github.com/zeroc-ice/ice/issues/1502
+
+- Removed UWP support https://github.com/zeroc-ice/ice/issues/1503
```
Comment (externl) [inline CHANGELOG-3.7.md:107]:
```suggestion
- Removed UWP support. (https://github.com/zeroc-ice/ice/issues/1503)
```

**Review by externl — COMMENTED** [review]
Looks good. Periods are missing for most sentences.

File: CHANGELOG-3.7.md [hunk 2]
```diff
@@ -8,87 +8,118 @@ We recommend that you use the release notes as a guide for migrating your
...
+  - [Python Changes](#python-changes-8)
   - [Ruby Changes](#ruby-changes-4)
 
+# Changes in Ice 3.7.10
+
+These are the major changes since Ice 3.7.9.
+
+## C++ Changes
+
+- Fixes failures building from source with G++ 13  https://github.com/zeroc-ice/ice/issues/1489
```
[the remark this answers is not in the document]
Comment (InsertCreativityHere) [reply CHANGELOG-3.7.md]: I agree, I think putting them in parenthesis would be clearer.
Also though, these are still sentences, so I think they should end with periods, like we do in the rest of the changelog:
```
Fixes failures building from source with G++ 13. (https://github.com/zeroc-ice/ice/issues/1489)
```

File: CHANGELOG-3.7.md [hunk 3]
```diff
@@ -8,87 +8,119 @@ We recommend that you use the release notes as a guide for migrating your
...
+- Fixed build failures with G++ 13. (https://github.com/zeroc-ice/ice/issues/1489)
+
+- Fixed the C++11 library names produced by Windows builds. (https://github.com/zeroc-ice/ice/issues/1502)
+
+- Removed UWP support. (https://github.com/zeroc-ice/ice/issues/1503)
+
+## C# Changes
+
+- Fix generated code for compatability with .NET 8. (https://github.com/zeroc-ice/ice/issues/1548)
+- ```
```
Comment (InsertCreativityHere) [inline CHANGELOG-3.7.md:112]: Extra backticks
```suggestion
```

File: CHANGELOG-3.7.md [hunk 4]
```diff
@@ -8,87 +8,119 @@ We recommend that you use the release notes as a guide for migrating your
...
+
+- Fixed build failures with G++ 13. (https://github.com/zeroc-ice/ice/issues/1489)
+
+- Fixed the C++11 library names produced by Windows builds. (https://github.com/zeroc-ice/ice/issues/1502)
+
+- Removed UWP support. (https://github.com/zeroc-ice/ice/issues/1503)
+
+## C# Changes
+
+- Fix generated code for compatability with .NET 8. (https://github.com/zeroc-ice/ice/issues/1548)
```
Comment (InsertCreativityHere) [inline CHANGELOG-3.7.md:111]: Looking at the rest of the changelog, we always write these in past-tense:
```suggestion
- Fixed generated code to be compatible with .NET 8. (https://github.com/zeroc-ice/ice/issues/1548)
```

File: CHANGELOG-3.7.md [hunk 5]
```diff
@@ -8,87 +8,119 @@ We recommend that you use the release notes as a guide for migrating your
...
+
+## Java Changes
+
+- Build fixes for compatibility with JDK 21
+
+- Fix generated Java constructors to correctly count the max parameters https://github.com/zeroc-ice/ice/issues/1540
+
+## Python Changes
+
+- Fix Python 3.12 build failures https://github.com/zeroc-ice/ice/issues/1558
```
Comment (InsertCreativityHere) [inline CHANGELOG-3.7.md:122]:
```suggestion
- Fixed Python 3.12 build failures. (https://github.com/zeroc-ice/ice/issues/1558)
```

File: CHANGELOG-3.7.md [hunk 6]
```diff
@@ -8,87 +8,119 @@ We recommend that you use the release notes as a guide for migrating your
...
+## C# Changes
+
+- Fix generated code for compatability with .NET 8. (https://github.com/zeroc-ice/ice/issues/1548)
+- ```
+
+## Java Changes
+
+- Build fixes for compatibility with JDK 21
+
+- Fix generated Java constructors to correctly count the max parameters https://github.com/zeroc-ice/ice/issues/1540
```
Comment (InsertCreativityHere) [inline CHANGELOG-3.7.md:118]:
```suggestion
- Fixed generated Java constructors to correctly count the max parameters. (https://github.com/zeroc-ice/ice/issues/1540)
```

**Review by externl — APPROVED** [review]
Looks good!

File: CHANGELOG-3.7.md [hunk 7]
```diff
@@ -8,87 +8,118 @@ We recommend that you use the release notes as a guide for migrating your
...
 
+# Changes in Ice 3.7.10
+
+These are the major changes since Ice 3.7.9.
+
+## C++ Changes
+
+- Fixed failures building from source with G++ 13. (https://github.com/zeroc-ice/ice/issues/1489)
+
+- Fixed the C++11 library names produced by Windows builds. (https://github.com/zeroc-ice/ice/issues/1502)
```
Comment (bernardnormier) [inline CHANGELOG-3.7.md:105]: I would remove this entry. It's a minor change, and as stated, confusing.

File: CHANGELOG-3.7.md [hunk 8]
```diff
@@ -8,87 +8,118 @@ We recommend that you use the release notes as a guide for migrating your
...
+
+- Removed UWP support. (https://github.com/zeroc-ice/ice/issues/1503)
+
+## C# Changes
+
+- Fixed generated code for compatability with .NET 8. (https://github.com/zeroc-ice/ice/issues/1548)
```
Comment (bernardnormier) [inline CHANGELOG-3.7.md:111]: compatibility ... .NET 8.0

File: CHANGELOG-3.7.md [hunk 9]
```diff
@@ -8,87 +8,118 @@ We recommend that you use the release notes as a guide for migrating your
...
+
+## Java Changes
+
+- Add support for JDK 21.
+
+- Fixed generated Java constructors to correctly count the max parameters. (https://github.com/zeroc-ice/ice/issues/1540)
```
Comment (bernardnormier) [inline CHANGELOG-3.7.md:117]: Should be rephrased. It's about not generating these constructors in some cases.

File: CHANGELOG-3.7.md [hunk 10]
```diff
@@ -8,87 +8,118 @@ We recommend that you use the release notes as a guide for migrating your
...
+
+- Removed UWP support. (https://github.com/zeroc-ice/ice/issues/1503)
+
+## C# Changes
+
+- Fixed generated code for compatability with .NET 8. (https://github.com/zeroc-ice/ice/issues/1548)
+
+## Java Changes
+
+- Add support for JDK 21.
```
Comment (InsertCreativityHere) [inline CHANGELOG-3.7.md:115]: For consistency with the rest of the file:
```suggestion
- Added support for JDK 21.
```
We always say "added support for X".

File: CHANGELOG-3.7.md [hunk 11]
```diff
@@ -8,87 +8,118 @@ We recommend that you use the release notes as a guide for migrating your
...
+
+- Fixed generated Java constructors to correctly count the max parameters. (https://github.com/zeroc-ice/ice/issues/1540)
+
+## Python Changes
+
+- Add support for Python 3.12. (https://github.com/zeroc-ice/ice/issues/1558)
```
Comment (InsertCreativityHere) [inline CHANGELOG-3.7.md:121]: Same
```suggestion
- Added support for Python 3.12. (https://github.com/zeroc-ice/ice/issues/1558)
```

**Review by InsertCreativityHere — APPROVED** [review]
Looks good!

**Pull request merged**